modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/js/ckeditor5/ClassicEditor.tsx

```diff
@@ -75,22 +87,26 @@ const ClassicEditor = ({config}: {config?: EditorConfig}) => {
 	}
 
 	return (
-		<CKEditor
-			config={{
-				...defaultConfig,
-				...config,
-			}}
-			editor={BaseClassicEditor}
-			onReady={(editor: BaseClassicEditor) => {
-				editor.ui.view.toolbar.items.map((item: any) => {
-					if (item.buttonView) {
-						item.buttonView.tooltipPosition = 'n';
-					}
+		<div className={`lfr-ck ${className ? className : ''}`}>
```

Comment:
Funnily enough, you cannot add a class to root wrapper of editor. See https://github.com/ckeditor/ckeditor5/issues/9539